packages/flutter/lib/src/widgets/safe_area.dart

```diff
@@ -70,6 +71,15 @@ class SafeArea extends StatelessWidget {
   /// The greater of the minimum insets and the media padding will be applied.
   final EdgeInsets minimum;
 
+  /// Specifies whether the [SafeArea] should maintain the [viewPadding] instead
```

Comment:
While technically correct, it's hard to understand what this sentence means...

Reply:
Maybe you can expand the paragraph at the bottom a little more to explain what it means that that padding is maintained, e.g. that content doesn't suddenly drop just because the keyboard popped up.

Reply:
I've updated this. Is the paragraph below a better explanation?

Reply:
That's better. I like it. Maybe add at the end that setting this to true will avoid the UI shift to be extra explicit.